composer.json

```diff
@@ -13,6 +13,23 @@
         "phpunit/phpunit": "^4.8",
         "fzaninotto/faker": "^1.8"
     },
+
```

Comment:
We need to be compatible from php v5.3.
We have Circle CI procedure to validate the installation, this procedure answers with this message:
nstalling dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for doctrine/instantiator 1.3.0 -> satisfiable by doctrine/instantiator[1.3.0].
    - doctrine/instantiator 1.3.0 requires php ^7.1 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 2
    - Installation request for phpdocumentor/reflection-common 2.0.0 -> satisfiable by phpdocumentor/reflection-common[2.0.0].
    - phpdocumentor/reflection-common 2.0.0 requires php >=7.1 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 3
    - Installation request for phpdocumentor/reflection-docblock 5.1.0 -> satisfiable by phpdocumentor/reflection-docblock[5.1.0].
    - phpdocumentor/reflection-docblock 5.1.0 requires ext-filter ^7.1 -> the requested PHP extension filter has the wrong version (0.11.0) installed.
  Problem 4
    - Installation request for phpdocumentor/type-resolver 1.1.0 -> satisfiable by phpdocumentor/type-resolver[1.1.0].
    - phpdocumentor/type-resolver 1.1.0 requires php ^7.2 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 5
    - Installation request for symfony/yaml v3.4.38 -> satisfiable by symfony/yaml[v3.4.38].
    - symfony/yaml v3.4.38 requires php ^5.5.9|>=7.0.8 -> your PHP version (5.3.29) does not satisfy that requirement.
  Problem 6
    - doctrine/instantiator 1.3.0 requires php ^7.1 -> your PHP version (5.3.29) does not satisfy that requirement.
    - phpunit/phpunit-mock-objects 2.3.8 requires doctrine/instantiator ^1.0.2 -> satisfiable by doctrine/instantiator[1.3.0].
    - Installation request for phpunit/phpunit-mock-objects 2.3.8 -> satisfiable by phpunit/phpunit-mock-objects[2.3.8].

Reply:
removed useless library